README.rst

```diff
@@ -216,12 +216,20 @@ See also the `related hardware ID settings in OpenWISP Controller
 Bootup Delay
 ------------
 
-The option ``bootup_delay`` can be used to make the agent wait for a random amount of seconds after the bootup of
-the device. Allowed random values range from 0 up to the value of ``bootup_delay``. The delay is applied only after the
-device has been registered.
+The option ``bootup_delay`` is used to delay the initialization of the agent
```

Comment:
What do you mean with the `initialization of the agent`? It's just a shell script with a `sleep` being executed - don't make it more complex than it is :grinning:

Reply:
I literally mean that we are delaying the initialization of the script, I am just trying to explain what we are doing by trying to use as less technical jargon as possible, if you have a better suggestion in mind please let me know @okraits.

Reply:
I literally mean that we are delaying the initialization of the script, I am just trying to explain what we are doing by trying to use as less technical jargon as possible, if you have a better suggestion in mind please let me know @okraits.

Reply:
@nemesisdesign I was just irritated because there is nothing like an initialization of a shell script, it is simply executed. I thought that the initial explanation, that the bootup delay makes the agent (the script) wait for a random amount of seconds, was quite simple and non-technical.

Reply:
@okraits we need to update this part because now it's going to be used by default, I think the initial explanation was not detailed enough because I often receive questions about it.

I don't think the use of the word initialization is a problem here because anyone can understand [what "initialize" means](https://dictionary.cambridge.org/dictionary/english/initialize), even if the agent is "just a script", it is still a program which runs indefinitely and needs to be started, it has a preparation sequence which we're delaying here with this feature. 

If you have ideas about how to improve this text please proceed to suggest here or let me know, thanks!

Reply:
@nemesisdesign Well, I have already told you that I thought that the initial explanation was fine but if you have received questions about it then it's understandable that you want to expand the explanation. I'm ok with that.